packaging/helm-charts/helm3/Makefile

```diff
@@ -0,0 +1,48 @@
+PROJECT_NAME=helm-charts
+
+include ../../../Makefile.os
+
+RELEASE_VERSION ?= latest
+CHART_NAME=strimzi-access-operator
+CHART_PATH ?= ../helm3/strimzi-access-operator
+CHART_RENDERED_TEMPLATES_TMP=../../../target/charts
+CHART_RENDERED_TEMPLATES_INSTALL=../../install
+HELM_CLI=helm
+
+helm_clean:
```

Comment:
Related to one of my other comments, maybe this should be defined in the top level Makefile, or at least we should reference this goal from the top level Makefile